src/xkit.cs

```diff
@@ -2090,4 +2093,78 @@ interface NSLayoutConstraint
 		[NullAllowed, Export ("identifier")]
 		string Identifier { get; set; }
 	}
+
+	[NoWatch]
+	[Mac (10,11)]
+	[MacCatalyst (13,0)]
+	[Model]
+	[Protocol]
+	[BaseType (typeof (NSObject))]
+	partial interface NSTextAttachmentContainer {
+		[Abstract]
+		[Export ("imageForBounds:textContainer:characterIndex:")]
+		[return: NullAllowed]
+#if MONOMAC && !XAMCORE_4_0
```

Comment:
Is the macOS target not going to work anymore in XAMCORE 4?

Looks like it would generate UIImage

Reply:
Or was this supposed to return `Image` as it's alias'd above

Reply:
Fixed to use the alias'ed definition above. Thanks!